graphbolt/CMakeLists.txt

```diff
@@ -207,8 +207,7 @@ if(USE_ROCM)
   target_link_libraries(${LIB_GRAPHBOLT_ROCM_NAME} PUBLIC "${TORCH_LIBRARIES}")
   target_link_libraries(${LIB_GRAPHBOLT_ROCM_NAME} PUBLIC graphbolt_gpu_cache)
 
-  # TODO(#23): Enable for perf. Was producing failing code
-  # target_compile_definitions(${LIB_GRAPHBOLT_ROCM_NAME} PRIVATE LIBCUDACXX_VERSION)
+  add_compile_definitions(LIBCUDACXX_VERSION)
```

Comment:
This isn't part of your PR (so also stupid GitHub won't let me comment on the actual lines), but why do we have to have a separate library for `graphbolt_gpu_cache` as well as including it in the extension for ROCm but for CUDA it's just included in the extension? I missed this previously.